crates/gpui/src/platform.rs

```diff
@@ -432,6 +432,7 @@ pub(crate) trait PlatformWindow: HasWindowHandle + HasDisplayHandle {
         Decorations::Server
     }
     fn set_app_id(&mut self, _app_id: &str) {}
+    fn map_window(&mut self) -> anyhow::Result<()> { Ok(()) } // We may want to set_app_id() first before map the window to display.
```

Comment:
Do we need this comment? It doesn’t seem useful.

Reply:
removed

Reply:
CI stalled. Can you trigger another run?